### PR TITLE
👔 `owner` of NFT income APIs refer to ISCN owner now

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

File: extractor/nft_test.go [hunk 1]
```diff
@@ -276,7 +276,7 @@ func TestSendNftWithPrice(t *testing.T) {
 	require.Len(t, incomeDetailsRes.IncomeDetails, 2)
 	require.Equal(t, nfts[0].ClassId, incomeDetailsRes.IncomeDetails[0].ClassId)
 	require.Equal(t, nfts[0].NftId, incomeDetailsRes.IncomeDetails[0].NftId)
-	require.Equal(t, ADDR_02_LIKE, incomeDetailsRes.IncomeDetails[0].Owner)
+	require.Equal(t, ADDR_01_LIKE, incomeDetailsRes.IncomeDetails[0].Owner)
```
Comment (williamchong) [inline extractor/nft_test.go:279]: This field is confusing since there is NftId right next to it, maybe we don't need Owner in response at all

Comment (WeiJun0827) [reply extractor/nft_test.go:279]: Yeah, probably don't have to include `owner` in response.

**Review by williamchong — APPROVED** [review]
Does ISCN id exists in nft class table now? seems we should actually denormalize it

**Comment by WeiJun0827** [comment]
> Does ISCN id exists in nft class table now? seems we should actually denormalize it

Do you mean the display table that Aurora is working on? Sounds it's preferred to include `iscnIdPrefix` in the response list?

**Pull request merged**